src/thread_cache.h

```diff
@@ -226,7 +226,7 @@ class ThreadCache {
     }
 
     void* Next() {
-      return SLL_Next(&list_);
+      return SLL_Next(list_);
```

Comment:
Hm, why you're reading list_ here. Looks wrong. I think original code is right, you want list_'s location to be treated as object and so then sll_next reads list_.